src/components/questionary/questionaryComponents/ProposalBasis/proposalBasisValidationSchema.ts

```diff
@@ -0,0 +1,25 @@
+import * as Yup from 'yup';
+
```

Comment:
Maybe this validation schema should be part of the validation package or what do you think?

Reply:
Great idea, although this requires a bigger refactor what I should definitely do maybe in the next sprint. I created a task describing a task
https://jira.esss.lu.se/browse/SWAP-1277